css/style.css

```diff
@@ -0,0 +1,96 @@
+#gb {
+  -webkit-app-region: drag; /* make titlebar draggable */
```

Comment:
This makes the search box, hamburger menu, and profile picture draggable.  We should revert to the default styles for those elements.

Reply:
what do you mean?

Reply:
If you click on the hamburger menu or search box and move your finger or mouse slightly, it will change the position of the window.  It should be setup so that only the blank part of the menu is draggable.

Reply:
I think that behaviour is actually ok. We'd limit the space where you can drag and I think the times where you accidentally click in the hamburger menu/search bar and drag the window seems quite low to me.

Reply:
I just tested it and I agree, it's actually very sensitive.

@bsramin can we make the elements in the title bar non-draggable please?